awscli/customizations/cloudtrail.py

```diff
@@ -204,8 +206,9 @@ def setup_new_bucket(self, bucket, prefix, policy_url=None):
         if policy_url:
             policy = requests.get(policy_url).text
         else:
-            data = self.s3.GetObject(bucket='awscloudtrail',
-                                     key=S3_POLICY_TEMPLATE)
+            data = self.s3.GetObject(
```

Comment:
It might be nice to provide a better error message if a policy hasn't been copied over to a specific region now that this is a new possible failure mode.